nix/nixos/tx-generator-service.nix

```diff
@@ -0,0 +1,94 @@
+(import ../. {}).commonLib.defServiceModule
```

Comment:
Wow that looks very polished.(maybe too much).
I guess this is more like a stub.
Basically it is adding yet an other layer to the configuration of the tx-generator.
The `tx-generator` itself will change and all the changes will propagate here.
OK but now we have it done anyway..

Reply:
Many nix-services do indeed have an option to just pass a custom configuration file (for example JSON) as configuration.
Would that not makes things simpler ?

Reply:
> Basically it is adding yet an other layer to the configuration of the tx-generator.

Are you familiar with NixOS service definitions?  This is one and is necessary for deployment.

Reply:
> Many nix-services do indeed have an option to just pass a custom configuration file (for example JSON) as configuration.
> Would that not makes things simpler ?

This is already implemented, indeed.